keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -466,7 +465,7 @@ to determine whether errors are related to this feature.
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
 
-Not yet, but will be done manually before transition to beta.
+This was done manually before transition to beta.
```

Comment:
Any findings? Can you be a bit more specific what exactly those manual tests did (e.g. what environment)?

Reply:
I used kubeadm to bring up a cluster, then enabled and disabled the feature in various components. While not strictly the same as downgrading a component, conceptually it is the same. Michelle said that this is okay.

While that worked as expected (pod cannot be created/scheduled/started depending on where the feature is off), one observation was that the error messages were sub-optimal for scheduler and kubelet: because the components ignored the unknown volume type, they reported something about "no volume plugin matched". I then changed error handling to be more useful: https://github.com/kubernetes/kubernetes/pull/99446/commits/edb9a8584c79376dd0ae17f46d79a6cc77c26fe7

Reply:
I also added something along those lines to the KEP and pushed an update.